src/test/java/org/opensearch/neuralsearch/query/HybridQueryAggregationsIT.java

```diff
@@ -100,43 +100,43 @@ protected boolean preserveClusterUponCompletion() {
 
     @SneakyThrows
     public void testPipelineAggs_whenConcurrentSearchEnabled_thenSuccessful() {
-        updateClusterSettings("search.concurrent_segment_search.enabled", true);
```

Comment:
Can we also do the same change by using CONCURRENT_SEGMENT_SEARCH_ENABLED in other places also like HybridSearchPostFilterIT?

Reply:
ack